backend/scandamus/players/views.py

```diff
@@ -170,4 +174,20 @@ def get_queryset(self):
         user = self.request.user
         player = Player.objects.get(user=user)
         return FriendRequest.objects.filter(to_user=player)
-    
\ No newline at end of file
+
+class AvatarUploadView(APIView):
+    authentication_classes = [JWTAuthentication]
+    permission_classes = [permissions.IsAuthenticated]
+    parser_classes = [MultiPartParser]
+
+    def put(self, request, format=None):
+        user = request.user
+        player = Player.objects.get(user=user)
+        avatar_file = request.FILES.get('avatar')
+
+        if avatar_file:
+            player.avatar = avatar_file
```

Comment:
```suggestion
            player.save()
            player.avatar = avatar_file
```
`player.save()` で何かしらの問題が起きた際も avatar url が切り替わってしまうので、逆にした方がいいかもしれません？